src/components/tooltip/Tooltip.tsx

```diff
@@ -285,7 +288,7 @@ class Tooltip extends React.Component<TooltipProps, State> {
         }
 
         if (showTooltip) {
-            if (!isLabelMatchingTooltipText || childAriaLabel === undefined) {
+            if (!id && (!isLabelMatchingTooltipText || childAriaLabel === undefined)) {
```

Comment:
why do we need this?

Reply:
To avoid duplicate aria-describeby labels